src/OptionResult.php

```diff
@@ -142,7 +142,7 @@ public function toArray()
         return $array;
     }
 
-    public static function create($specs, array $values = array(), array $arguments = null)
+    public static function create($specs, array $values = array(), array $arguments = [])
```

Comment:
Could you provide the reference about this deprecated message? Thanks.

Reply:
> Could you provide the reference about this deprecated message? Thanks.

https://php.watch/versions/8.4/implicitly-marking-parameter-type-nullable-deprecated